go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	code.cloudfoundry.org/clock v1.0.0 // indirect
 	collectd.org v0.5.0
 	github.com/Azure/azure-event-hubs-go/v3 v3.2.0
+	github.com/Azure/azure-pipeline-go v0.2.3 // indirect
```

Comment:
Where does this come from?

Reply:
go mod tidy added it. I'm not sure why

Reply:
Strange. I can live with this change if you can. ;-)